test/regression/sql/basic.sql

```diff
@@ -20,3 +20,8 @@ SET client_min_messages TO default;
 
 DROP TABLE t;
 DROP TABLE empty;
+
+-- Check that DROP / CREATE extension works
+
```

Comment:
I don't quite see how this is testing the problem, as it occurred from running `create extension pg_duckdb;` without having loaded the `pg_duckdb` extension, but never the less the changes fixed it 👍